.gitignore

```diff
@@ -0,0 +1,6 @@
+
+# Mac stuff
+*.DS_Store
+
+# R stuff
+uptasticsearch.Rcheck
```

Comment:
Let's also add more R stuff here such as history, data, Renviron, etc in case we accidentally pushed them here